frontend/packages/core/src/Assets/colors.tsx

```diff
@@ -0,0 +1,209 @@
+// COLORS FROM FIGMA
+// NOTE THAT THOSE WITH GRADIENTS ONLY HAVE THEIR BASES
+
+interface CLUTCH_PALETTE {
```

Comment:
wasn't sure if this should be capitalized like this

Reply:
We could rename it to `ClutchPalette` as well, might look a little cleaner.

One thing to make it a bit easier could be to split out the types from this file, so it doesn't get cluttered, and then type every key as well.

```
export interface ClutchPalette {
   NEUTRAL: NeutralColors;
   BLUE: BlueColors;
}

interface NeutralColors {
   5: string;
   10: string;
   ...
}
```

This way we have every possible value defined and if we want to add any new palettes it will require every single color be present before it will allow it.